### ISSUE TITLE
Use 

### ISSUE THREAD

**Issue by esdee** [segmentio/analytics.js#659]
I've noticed that in the [integrations-repo](https://github.com/segmentio/analytics.js-integrations) there is a [google-analytics-4](https://github.com/segmentio/analytics.js-integrations/tree/master/integrations/google-analytics-4) integration.

How would I use this from analytics.js - the integrations do not reference it [here](https://github.com/segmentio/analytics.js/blob/master/lib/integrations.js)

**Comment by dev-goformative** [comment]
Yes I also noticed it's not included in the components.json and has not been released to npm.  Is there a way to workaround this and use the GA4 integration, or will the GA4 integration be released to NPM sometime soon?
